samples/awssqs_source/README.md

```diff
@@ -10,7 +10,7 @@
 1.  If your installed version of Eventing did not include the in-memory channel
     provisioner, install the
     [in-memory `ClusterChannelProvisioner`](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel)
-    now. If you installed Eventing using the `release.yaml` file, the channel
+    now. If you installed Eventing using the `eventing.yaml` file, the channel
```

Comment:
Thanks for catching this -- looks like we were pointing at the wrong file since 0.4.0!

Reply:
**No, this change is incorrect.** `eventing.yaml` does NOT include the in-memory channel provisioner. `release.yaml` does. https://github.com/knative/eventing/blob/18f7b078bdd9ea70a2b4db88abbedc321df0ed5c/hack/release.sh#L32